app/controllers/records_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RecordsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @category = current_user.categories.find(params[:category_id])
+    @records = @category.records
+  end

```

Comment:
- [ ] Your transactions on the page should be displayed based on the most recent record. So for example if I have 2 records (record 1 and record 2), the page should display the records in this order: `record 2`, `record 1` based on the most recent transaction.